matrix-sdk-android/src/androidTest/java/org/matrix/android/sdk/common/CommonTestHelper.kt

```diff
@@ -83,6 +95,15 @@ class CommonTestHelper(context: Context) {
         return logIntoAccount(userId, TestConstants.PASSWORD, testParams)
     }
 
+    fun cleanUpOpenedSessions() {
+        trackedSessions.forEach {
+            runBlockingTest {
+                it.signOutService().signOut(true)
```

Comment:
out of interest, which state causes trouble that requires a sign out?

Reply:
You mean that it should not be necessary to sign out at the end of the test?

Reply:
yep, ideally each test would create its own `session` instance (maybe creating new accounts as well?)

was wondering if it's the database tables that are being shared or if there's other static state~

Reply:
- there is one DB per session for cache and one DB per session for crypto.
- auth data is in a dedicated DB (so sort of shared)